src/FluidXPlate.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mll\LiquidHandlingRobotics;
+
+use Mll\Microplate\Coordinate;
+use Mll\Microplate\CoordinateSystem96Well;
+use Mll\Microplate\Microplate;
+
+/**
+ * @phpstan-extends Microplate<string, CoordinateSystem96Well>
+ */
+class FluidXPlate extends Microplate
+{
+    public const FLUIDX_BARCODE_REGEX = /* @lang RegExp */ '/[A-Z]{2}(\d){8}/';
+
+    public string $rackId;
+
+    public function __construct(string $rackId)
+    {
+        $this->rackId = $rackId;
+        if (! (bool) \Safe\preg_match(self::FLUIDX_BARCODE_REGEX, $rackId)) {
+            throw new TecanScanException('Invalid rack id:' . $this->rackId);
+        }
```

Comment:
```suggestion
        if (0 === \Safe\preg_match(self::FLUIDX_BARCODE_REGEX, $rackId)) {
            throw new TecanScanException("Invalid rack ID: {$rackId}");
        }
        $this->rackId = $rackId;
```

- Validate first
- Clarify expected results from preg_match
- Avoid validating `$rackId` and throwing on `$this->rackId`
- proper spacing in message